### DISCUSSION TITLE
lamellar orientation

### DISCUSSION THREAD

**Discussion by kittygg** [neperfepx/neper#529] [Q&A] [answered]
Hi all,

How can the orientation of the lamellar be controlled?  Such as the inclination angle of lamellars is 10 degrees or 90 degrees.

neper -T -n from_morpho -dim 2 -morpho "lamellar(w=1:2)" -statcell "scaleid(1),lam" -o lam

Thanks!

**Comment by rquey** [comment, marked as answer]
Use the `v` argument of `lamellar`, see [-morpho](https://neper.info/doc/neper_t.html?highlight=lamellar#cmdoption-morpho).
